## Pipeline to identify candidate intergenic transcribed regions in the yeast Saccharomyces cerevisiae.

To run the pipeline you need first to edit the *config.txt* file with paths to different required annotation files and bam files. The input bam files used by this pipeline were generated by mapping RNAseq data to the reference yeast genome.
The *data* folder contains predicted transcription start sites ([ScerYPDconsensusClusters.gff](data/ScerYPDconsensusClusters.gff)) from the YeasTSS database (http://www.yeastss.org/), annotated 5' and 3' UTRs ([Nagalakshmi_2008_5_3_UTRs_V64.bed](data/Nagalakshmi_2008_5_3_UTRs_V64.bed)) and yeast chromosome sizes ([chromosome_sizes.txt](data/chromosome_sizes.txt)).

To run the pipeline simply use the following command:

```bash
RunStringtie.sh
```

To load pipeline dependencies with *conda* run the following command:

```bash
conda env create -f environment.yml
```

### Dependencies

- Perl 5

- bedtools v2.31.1

- igvtools v2.17.3

- stringtie v2.2.1

- ucsc-wigtobigwig v366
